src/components/dataset/data/sidebar/filter/FreetextFilter.jsx

```diff
@@ -35,54 +33,20 @@ export class FreetextFilter extends React.PureComponent {
     column: PropTypes.object,
     dataset: PropTypes.object,
     filterMap: PropTypes.object,
-    filterStatement: PropTypes.string,
     handleChange: PropTypes.func,
     handleFilters: PropTypes.func,
-    joinStatement: PropTypes.string,
-    options: PropTypes.array,
-    originalValues: PropTypes.object,
     table: PropTypes.string,
     tableName: PropTypes.string,
     toggleExclude: PropTypes.func,
-    token: PropTypes.string,
-    values: PropTypes.object,
-  };
-
-  transformResponse = (response) => {
-    const options = [];
-    if (response) {
-      // eslint-disable-next-line
-      response.map((r) => {
-        const name = r.f[0].v;
-        options.push(name);
-      });
-    }
-    return options;
+    values: PropTypes.arrayOf(PropTypes.object),
   };
 
   onInputChange = async (event) => {
-    const { column, dataset, tableName, token, filterStatement, joinStatement } = this.props;
-    const { bq } = this.state;
     const { value } = event.target;
 
     this.setState({
       inputValue: value,
     });
-
-    const response = await bq.getAutocompleteForColumnDebounced(
-      value,
-      column.name,
-      dataset,
-      tableName,
-      token,
-      filterStatement,
-      joinStatement,
-    );
-
-    const transformedResponse = this.transformResponse(response);
-    this.setState({
-      options: transformedResponse,
-    });
```

Comment:
I don't think this is necessary. 
By filtering down the responses, we remove the ability to add more options after filtering. 
Let's say there are 3 options: 'a' 'b', & 'c'. (This is a bad example because it would actually use the checkbox option, which DOES keep the original values in the list even after filtering with an updated count according to the filter set. Anyway...)
* let's say the user selects 'a' and then hits "apply filter"
* Then, the user decides they also want to include 'b'. If we kept this code in place, the the option 'b' would no longer be present. Yes, they could just type in 'b' if they knew what they were looking for, but otherwise, they would have to clear the filter and then go back and select 'a' and 'b'.

Reply:
It should only ever hit BQ (via API) on first load and on actually hitting the "apply filter" button. With this, do you think we're at risk for hammering the BQ?